src/loaders/ObjectLoader.js

```diff
@@ -610,6 +617,94 @@ THREE.ObjectLoader.prototype = {
 
 			}
 
+			/*
+			 * Now SkinnedMesh could have two bone sets in its children,
+			 * one is made from geometry.bones in SkinnedMesh constroctor
+			 * and another one is made from data.children in this method.
+			 * These two bone sets correspond to the same bones.
+			 * The former would represent a default stand pose while
+			 * the latter would represent a pose updated by skinning.
+			 * To let SkinnedMesh be updated pose, here this logic copies
+			 * the latter one to the former one then removes the latter one.
+			 * If in the latter one there're objects which don't correspond to
+			 * the ones in the former one, they'll be moved to the former one.
+			 */
```

Comment:
So it comes down to what the intended scope of Three.js and the editor is supposed to be.

Authoring software will most probably do a better job when it comes to editing, OTOH I guess it could be very useful (e.g. for debugging) to have a GUI-way to explore an exported animation, maybe even to change it in some ways.

Looking at all the trouble with the exporters lately, going "baked only" may make it more difficult to get right even if it means fewer code, but I could be wrong.